featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -266,7 +268,8 @@ def _filter_features(self, features):
 
         return f_keep
 
-    def _run_dfs(self, entity, entity_path, all_features, max_depth):
+    def _run_dfs(self, entity, entity_path, all_features, max_depth,
+                 allowed_variable_types=None):
```

Comment:
once we remove `allowed_variable_types` from `_build_forward_features`, we no longer need this parameter to `_run_dfs`